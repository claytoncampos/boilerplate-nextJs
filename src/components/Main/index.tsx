const Main = () => (
  <main>
    <h1>Next Avan√ßado</h1>
  </main>
)

export default Main
