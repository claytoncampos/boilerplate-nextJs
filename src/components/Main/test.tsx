import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main>', () => {
  it('sloud render the heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /next avan√ßado/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
