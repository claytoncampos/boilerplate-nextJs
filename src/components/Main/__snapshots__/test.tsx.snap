// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main> sloud render the heading 1`] = `
<main
  class="styles__Wrapper-dzx1ao-0 bjFRYl"
>
  <img
    alt="Imagem de um átomo e React avançado escrito ao lado."
    class="styles__Logo-dzx1ao-1 dcISKB"
    src="/img/logo.svg"
  />
  <h1
    class="styles__Title-dzx1ao-2 kSTVFj"
  >
    React Avançado
  </h1>
  <h2
    class="styles__Description-dzx1ao-3 dsoDmA"
  >
    TypeScript, ReactJS, NextJS e Styled Components
  </h2>
  <img
    alt="Um desenvolvedor de frente para a tela com código"
    class="styles__Ilustration-dzx1ao-4 brcvLB"
    src="/img/hero-illustration.svg"
  />
</main>
`;
